css/style.css

```diff
@@ -0,0 +1,134 @@
+
+* {
+  box-sizing: border-box;
+  padding:0px ;
+  margin: 0px;
+}
+body
+{
+  display: flex;
+  flex-wrap: wrap;
+}
+
+body {
```

Comment:
Mistakenly, two `body` selectors were declared previously, can you combine the rules into a single `body` selector.

Reply:
okay, I am on it.